UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/Pipenets/Pipe.cs

```diff
@@ -33,6 +33,7 @@ public void WrenchAct()
 		if (anchored)
 		{
 			anchored = false;
+			spriteRenderer.sortingLayerID = SortingLayer.NameToID("Items");
```

Comment:
you could cache these and make static, but it’s a minor optimisation

Reply:
yeah I kinda want to copy all layers from tg too

Reply:
actually just an idea because I dunno how they are defined